experimental/vertx/image/Dockerfile-stack

```diff
@@ -0,0 +1,38 @@
+# Dockerfile for building the stack
```

Comment:
Just out of curiosity, are we not watching for any file changes?

Reply:
That is done by the maven plugin